### PR TITLE
Grw-1759 - 🚗  🇸🇪  Registration field

### PR DESCRIPTION
**Pull request by KarnellSchultz**
## Describe your changes

Created a registration field component

https://user-images.githubusercontent.com/50870173/201635664-8ee2bfcd-f465-4236-b137-35cd6ef27cde.mov

## Justify why they are needed

Registration numbers are needed for cars. This specific field is going to help users with formatting registration numbers.

## Checklist before requesting a review

- [x] I have performed a self-review of my code

[GRW-1759]: https://hedvig.atlassian.net/browse/GRW-1759?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by guilhermespopolin — APPROVED** [review]
Noice 👏🏻  
I guess I'd only rename it to something more specific like `CarRegistrationNumberField`. I know it's a big name but `RegistrationField` might be too generic.

**Comment by KarnellSchultz** [comment]
@guilhermespopolin Tack - I don't mind a long name at all. I'll change it 👍

File: apps/store/src/components/PriceForm/CarRegistrationField.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  return (
+    <>
+      <input
+        type="text"
+        name={field.name}
+        required={field.required}
+        value={value}
+        readOnly
+        hidden
```
Comment (KarnellSchultz) [inline apps/store/src/components/PriceForm/CarRegistrationField.tsx:34]: ended up needing this to send the correct `name` to the Form component / backend

Comment (alebedev) [reply apps/store/src/components/PriceForm/CarRegistrationField.tsx:34]: Was anything wrong with removing -visible-input suffix from InputField.name?

Comment (KarnellSchultz) [reply apps/store/src/components/PriceForm/CarRegistrationField.tsx:34]: That works as well. Removing the hidden field :check:

**Pull request merged**